backend/api/tests/views/test_views_task_profiling.py

```diff
@@ -169,6 +153,41 @@ def test_task_profiling_add_step_no_datetime_change(authenticated_backend_client
     assert new_datetime == previous_datetime
 
 
+@override_settings(**ORG_SETTINGS)
+@pytest.mark.django_db
+def test_already_exist_step(authenticated_backend_client, task_profiling):
+    url = reverse("api:step-list", args=[str(task_profiling.compute_task.key)])
+    content = {"step": "step name", "duration": datetime.timedelta(seconds=20)}
+    response = authenticated_backend_client.post(url, content, **EXTRA)
+    assert response.status_code == status.HTTP_201_CREATED
+
+    response = authenticated_backend_client.post(url, content, **EXTRA)
+    assert response.status_code == status.HTTP_409_CONFLICT
```

Comment:
I'm not sure to understand what we want to do with the 409. Will we just not display anything? will it be caught somewhere else and reconciled later?

Reply:
Cf `_send_profiling_event`